src/store.js

```diff
@@ -113,6 +125,32 @@ export const actions = {
   setProductionMode({ commit }, payload) {
     commit('SET_PRODUCTION_MODE', payload);
   },
+  setContextShowDropdown({ commit }, payload) {
+    commit('SET_CONTEXT_DROPDOWN', payload);
+  },
+  setGraphMode({ commit }, payload) {
+    commit('SET_GRAPH_MODE', payload);
+  },
+  setFullscreen({ commit, state, dispatch }) {
+    const body = document.querySelector('body');
+    dispatch('setContextShowDropdown', false);
+
+    if (state.fullscreen) {
+      commit('SET_FULLSCREEN', false);
+      body.style.height = 'auto';
+      body.style.overflow = 'auto';
+    } else {
+      commit('SET_FULLSCREEN', true);
+      body.style.height = '100vh';
+      body.style.overflow = 'hidden';
+
+      this._vm.$ga.event({
+        eventCategory: 'Card',
+        eventAction: 'Open fullscreen',
+        eventLabel: this.graphMode,
+      });
```

Comment:
We might want to see if there's a way to track which card (and graphMode?) the fullscreen is triggered on? Not important though.